agent/agent_endpoint.go

```diff
@@ -1188,12 +1189,16 @@ func (s *HTTPServer) AgentToken(resp http.ResponseWriter, req *http.Request) (in
 
 	// Figure out the target token.
 	target := strings.TrimPrefix(req.URL.Path, "/v1/agent/token/")
+	triggerAntiEntropySync := false
 	switch target {
 	case "acl_token", "default":
 		s.agent.tokens.UpdateUserToken(args.Token, token_store.TokenSourceAPI)
```

Comment:
Should we trigger it for updates to the default token as these could be used for service registrations?

Reply:
Done.